tools/NuGet/template-nuget/DynamoVisualProgramming.Core.nuspec

```diff
@@ -10,16 +10,15 @@
         <icon>content\images\logo.png</icon>
         <requireLicenseAcceptance>false</requireLicenseAcceptance>
         <description>Assemblies required to start a DynamoModel and execute DesignScript code. Contains the following files:
-        1) DSIronPython.dll
-        2) DynamoApplications.dll
-        3) DynamoCore.dll
-        4) DynamoInstallDetective.dll
-        5) DynamoShapeManager.dll 
-        6) DynamoUtilities.dll
-        7) ProtoCore.dll
-        8) DesignScriptBuiltin.dll
-        9) VMDataBridge.dll
-        10) DSCPython.dll</description>
+        1) DynamoApplications.dll
+        2) DynamoCore.dll
+        3) DynamoInstallDetective.dll
+        4) DynamoShapeManager.dll 
+        5) DynamoUtilities.dll
+        6) ProtoCore.dll
+        7) DesignScriptBuiltin.dll
+        8) VMDataBridge.dll
+        9) DSCPython.dll</description>
```

Comment:
@pinzart90 to support Civil with minimal changes does it make sense to keep DSCPython here?

Reply:
Well..we never set out to get rid of DSCPython for 2.13
Keeping it around will provide at least 1 Pyuthon engine out of the box.
And yes...it will allow CIVIL and ALIAS to work with not changes in API